sort/sort.html

```diff
@@ -47,18 +47,16 @@
             </form>
         </div>
 
-        <canvas id="myChart"></canvas>
+        <canvas id="myChart"></canvas> </canvas>
         <script>
-            
-            const ctx = document.getElementById('myChart').getContext('2d');
-            const xlabels = [];
-            const myChart = new Chart(ctx, {
+            var ctx = document.getElementById('myChart');
+            var myChart = new Chart(ctx, {
                 type: 'bar',
                 data: {
-                    labels: [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15],
+                    labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
```

Comment:
You can't use colors to identify each bar, when the bar changes the vlaues 'Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange' are going to be deleted.

Reply:
That's was your code